package.json

```diff
@@ -14,7 +14,7 @@
     "build-storybook": "cd packages/superset-ui-demo && yarn build-storybook",
     "chromatic": "cd packages/superset-ui-demo && yarn chromatic",
     "sb": "yarn storybook",
-    "clean": "rm -rf ./{packages,plugins}/**/{lib,esm,tsconfig.tsbuildinfo} build/",
+    "clean": "yarn build --clean --no-babel --no-type",
```

Comment:
It's because `clean` doesn't clean up `node_modules` under each package installed by `npm link`. I tried to do some optimization in this PR: https://github.com/apache-superset/superset-ui/pull/862/files#diff-a992dd64a8c626fedc95d613f43602b91f6dc6339893afaf405dc05fa13c9cea

But it may not fully resolve this issue still.